app/models/reports.rb

```diff
@@ -1,10 +1,19 @@
 module Reports
+  REGISTRATION_BUFFER_IN_MONTHS = 3
+  MAX_MONTHS_OF_DATA = 24
+  PERCENTAGE_PRECISION = 0
+
   # The default period we report on is the current month.
   def self.default_period
     Period.month(Time.current.in_time_zone(Period::REPORTING_TIME_ZONE))
   end
 
-  REGISTRATION_BUFFER_IN_MONTHS = 3
-  MAX_MONTHS_OF_DATA = 24
-  PERCENTAGE_PRECISION = 0
+  def self.reporting_schema_v2?
```

Comment:
Consolidating the flag here, as its global (per request) state and trying to minimize the pain from it.